airflow/providers/odbc/hooks/odbc.py

```diff
@@ -69,20 +69,20 @@ def __init__(
         self._connect_kwargs = connect_kwargs
 
     @property
-    def connection(self):
+    def connection_(self):
```

Comment:
```suggestion
    def connection(self):
```
You can overwrite it.